DotNetTry.sln

```diff
@@ -51,6 +51,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MLS.HumanizerRunner", "MLS.
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PackageTool.Tests", "PackageTool.Tests\PackageTool.Tests.csproj", "{11FDD0E8-D07B-41C9-AF7E-E7F735D91ECF}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.DotNet.Try.Template", "Microsoft.DotNet.Try.Template\content\Microsoft.DotNet.Try.Template.csproj", "{9395C81D-C557-4681-8C4C-D5C1F160CD58}"
```

Comment:
Re: the project name, will additional templates go into this same project?

Also, "project template" might be more informative than "template".